src/ZoneServer/Database/ZoneDb.cs

```diff
@@ -825,7 +825,7 @@ protected void LoadVars(MySqlConnection conn, Variables vars, string tableName,
 		private void LoadChatMacros(Account account)
 		{
 			using (var conn = this.GetConnection())
-			using (var mc = new MySqlCommand("SELECT * FROM `chatmacros` WHERE `accountId` = @accountId ORDER BY `index` DESC", conn))
+			using (var mc = new MySqlCommand("SELECT * FROM `chatMacros` WHERE `accountId` = @accountId ORDER BY `index` DESC", conn))
```

Comment:
Two problems: First of all, you should ideally not put random SQL fixes into a skill PR. Second, this already was fixed (for all new users) by fixing the SQL files/table names. This change would actually unfix the problem^^"

Reply:
Wait, how the heck did that get in there?  This is definitely some kind of merging glitch, there's no ZoneDB changes for this PR.